### PR TITLE
Add schedule page

### PR DESCRIPTION
**Pull request by razvand**
Add schedule page to be integrated / linked in other pages.

### CODE REVIEW

File: content/en/docs/sessions/schedule.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+* Wednesday, August 25, 2021
+  * 4pm-4:30pm CEST: Tech Talk (Simon)
+  * 4:30pm-8pm CEST: Session 03: Benchmarking and Debugging (trainers: CristiV, RazvanD)
+* Thursday, August 26, 2021
+  * 4pm-8pm CEST: Session 04: Complex Applications (trainers: Costi, Vlad)
+* Friday, August 27, 2021
+  * 4pm-4:30pm CEST: Tech Talk (Ștefan)
+  * 4:30pm-8pm CEST: Session 05: Contributing to Unikraft (trainers: Vlad, RazvanD)
+* Monday, August 30, 2021
+  * 4pm-8pm CEST: Session 06: Testing Unikraft (trainers: CristbiB, Alex)
```
Comment (cristian-vijelie) [inline content/en/docs/sessions/schedule.md:27]:
```suggestion
  * 4pm-8pm CEST: Session 06: Testing Unikraft (trainers: CristiB, Alex)
```

**Review by cristian-vijelie — COMMENTED** [review]
I would also use the full mane for the trainers, considering that it's a public schedule

File: content/en/docs/sessions/schedule.md (hunk 1 above)
Comment (razvand) [reply content/en/docs/sessions/schedule.md]: @cristian-vijelie , I'm fond of keeping it more informal. I think only the first name makes it for familiar. @nderjung , what are your thoughts on this?

File: content/en/docs/sessions/schedule.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+date: 2021-08-19
+---
+
+Unikraft Summer of Code 2021 (USoC'21) consists of 10 sessions in 10 days and a hackaton.
+Each session is 4 hours long and consists of practical demos and then exercises for participants.
+The hackathon is an 8 hours even where you'll have different tasks to enable, test, fix, evaluate and improve applications and libraries with Unikraft.
+
+The complete schedule for USoC'21 is (all times in CEST - Central European Summer Time):
+
+| Date | Interval | Activity | Overseer(s) |
```
Comment (nderjung) [inline content/en/docs/sessions/schedule.md:13]: "Overseer(s)" -> "TA(s)"

File: content/en/docs/sessions/schedule.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+| Mon, 23.08.2021 | 3:30pm-4pm<br/> 4pm-8pm| Opening Ceremony<br/> [Session 01: Baby Steps]({{<ref "/docs/sessions/01-baby-steps" >}} "01. Baby Steps") | RazvanD, Alex, CristiV <br/> Alex, CristiV, Vlad |
+| Tue, 24.08.2021 | 4pm-8pm | [Session 02: Behind the Scenes]({{<ref "/docs/sessions/02-behind-scenes" >}} "02. Behind the Scenes") | RazvanD, Costi |
+| Wed, 25.08.2021 | 4pm-4:30pm<br/> 4:30pm-8pm | Tech Talk: Debugging and Tracing in Unikraft<br /> [Session 03: Benchmarking and Debugging]({{<ref "/docs/sessions/03-benchmarking-and-debugging" >}} "03. Benchmarking and Debugging") | Simon<br /> CristiV, RazvanD |
+| Thu, 26.08.2021 | 4pm-8pm | [Session 04: Complex Applications]({{<ref "/docs/sessions/04-complex-applications" >}} "04. Complex Applications") | Costi, Vlad |
+| Fri, 27.08.2021 | 4pm-4:30pm<br/> 4:30pm-8pm | Tech Talk<br/> [Session 05: Contributing to Unikraft]({{<ref "/docs/sessions/05-contributing-to-unikraft" >}} "05. Contributing to Unikraft") | Ștefan<br/> Vlad, RazvanD |
+| Mon, 30.08.2021 | 4pm-8pm | [Session 06: Testing Unikraft]({{<ref "/docs/sessions/06-testing-unikraft" >}} "06. Testing Unikraft") | CristbiB, Alex |
```
Comment (cristian-vijelie) [inline content/en/docs/sessions/schedule.md:20]:
```suggestion
| Mon, 30.08.2021 | 4pm-8pm | [Session 06: Testing Unikraft]({{<ref "/docs/sessions/06-testing-unikraft" >}} "06. Testing Unikraft") | CristiB, Alex |
```

File: content/en/docs/sessions/schedule.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+---
+title: "Schedule"
+linkTitle: "Schedule"
+date: 2021-08-19
+---
+
+Unikraft Summer of Code 2021 (USoC'21) consists of 10 sessions in 10 days and a hackaton.
+Each session is 4 hours long and consists of practical demos and then exercises for participants.
+The hackathon is an 8 hours even where you'll have different tasks to enable, test, fix, evaluate and improve applications and libraries with Unikraft.
```
Comment (cristian-vijelie) [inline content/en/docs/sessions/schedule.md:9]:
```suggestion
The hackathon is an 8 hours event where you'll have different tasks to enable, test, fix, evaluate and improve applications and libraries with Unikraft.
```

**Review by cristian-vijelie — APPROVED** [review]
Apart from one (new) typo, everything looks fine to me.

File: content/en/docs/sessions/schedule.md (hunk 1 above)
Comment (nderjung) [reply content/en/docs/sessions/schedule.md]: First names are fine, since we are a small group of people

**Pull request merged**